runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -52,6 +56,17 @@
         implements IClientRouter {
 
     /**
+     * Metrics: meter (counter), histogram
+     */
+    private static Gauge<Integer> gaugeConnected;
+    private static Timer timerConnect;
+    private static Timer timerSyncOp;
+    private static Counter counterConnectFailed;
+    private static Counter counterSendDisconnected;
+    private static Counter counterSendTimeout;
+    private static Counter counterAsyncOpSent;
```

Comment:
I think that we actually agreed that all meter references will be static. They are in fact global, and referred to by a unique name. So I guess references might as well be global.

By comparison, individual sl4fj loggers are typically static.

(it will work correctly either way, but I believe that in terms of style, we converged on static reference.)